### DISCUSSION TITLE
DoHome Homekit SmartBulb Tasmota?

### DISCUSSION THREAD

**Discussion by darellsison** [arendst/Tasmota#11251] [Q&A] [answered]
Hoping to get some info on this smart bulb. I was really hoping to get this converted to tasmota for a non-cloud integration into homebridge.

Sadly I dont see any info on it on supported devices.

Thanks in advance!

**Comment by Jason2866** [comment]
Sorry, i dont get what are you asking. Maybe you can provide a link?

**Reply by darellsison** [reply]
Thank you for the response.

I am hoping to change the firmware of this to Tasmota.

https://m.aliexpress.com/i/4000153608214.html

Its normal firmware gets it into homekit, but it wont work when its on my separate iot network hence my thinking of removing it completely off the cloud and purely local would work better.

I have been searching for instructions and worry that it might not be supported by Tasmota.

Thanks again.

**Comment by Jason2866** [comment, marked as answer]
Since it is not mentioned in the seller description of the used chip inside, we cant tell you.
For a chance to get Tasmota onto the device there has to be a ESP8266 inside and there is the possibility to do wired flashing
(needs reversible disassemble of the device). There is no OTA possibility.

**Comment by wiresio** [comment]
The DoHome bulb has a [DT-Light ESP8285 Lighting Module (DMP-L1)](https://templates.blakadder.com/DMP-L1.html). To flash the bulb you have to disassemble it, solder cables, flash, and then reassemble everything.
![IMG_8919](https://github.com/arendst/Tasmota/assets/4670534/f87204b2-7095-4942-bfbc-bc68a5f435c5)

**Comment by wiresio** [comment]
Another alternative might be this: https://hackaday.io/project/167470-how-to-flash-dohome-iot-devices-to-alternative-fir/details
